tests/PluginTest.php

```diff
@@ -93,16 +93,12 @@ protected function setUp(): void
 
         public function test_it_implements_plugin_interface(): void
         {
-            $plugin = new Plugin();
-
-            self::assertTrue(is_a($plugin, PluginInterface::class));
+            self::assertInstanceOf(PluginInterface::class, new Plugin());
```

Comment:
Using the `new Plugin()` to create class instance to do assert instance and it can avoid following PHPStan detecting message:

```
 ------ -------------------------------------------------------------------- 
  Line   tests/PluginTest.php                                                
 ------ -------------------------------------------------------------------- 
  98     Call to static method PHPUnit\Framework\Assert::assertInstanceOf()  
         with 'Composer\\Plugin\\PluginInterface' and                        
         Infection\ExtensionInstaller\Plugin will always evaluate to true.   
  105    Call to static method PHPUnit\Framework\Assert::assertInstanceOf()  
         with 'Composer\\EventDispatcher\\EventSubscriberInterface' and      
         Infection\ExtensionInstaller\Plugin will always evaluate to true.   
 ------ -------------------------------------------------------------------- 
```